File: app/models/message.rb
class Message < ActiveRecord::Base
  attr_accessible :body, :recipient_id, :sender_id, :read

  belongs_to :sender,
  	:class_name => "User",
  	:foreign_key => :sender_id

  belongs_to :recipient,
  	:class_name => "User",
  	:foreign_key => :recipient_id
end

File: app/assets/javascripts/models/message.js
Fallinlike.Models.Message = Backbone.RelationalModel.extend({
  relations: [{
    type: Backbone.HasOne,
    key: "sender",
    keySource: "sender_id",
    keyDestination: "sender_id",
    relatedModel: "Fallinlike.Models.User",
    autoFetch: true,
    includeInJSON: "id"
  },
  {
    type: Backbone.HasOne,
    key: "recipient",
    keySource: "recipient_id",
    keyDestination: "recipient_id",
    relatedModel: "Fallinlike.Models.User",
    autoFetch: true,
    includeInJSON: "id"
  }],

  urlRoot: "/messages/"
});